client/state/plugins/marketplace/types.ts

```diff
@@ -3,10 +3,26 @@
  */
 import type { Action } from 'redux';
 
+export enum MARKETPLACE_ASYNC_PROCESS_STATUS {
```

Comment:
Any reason to prefer an enum to a union here?

Reply:
No specific reason just felt it as a good way to defined a set of enumerated values. Any disadvantages in using enums over unions?

Reply:
Unions are generally preferred over enums for several reasons, I see some good rationales are written here: https://stackoverflow.com/questions/40275832/typescript-has-unions-so-are-enums-redundant

I don't have a strict opinion, it's just the verbosity that bugs me a little bit when using enums 😉